tests/testthat/test-xml_nodes_to_lints.R

```diff
@@ -110,14 +110,13 @@ test_that("it doesn't produce invalid lints", {
         range_start_xpath = xp_invalid,
         range_end_xpath = xp_range_end
       )
-    },
-    rex::rex("Could not find range start for lint. Defaulting to start of line.")
```

Comment:
If we are committed to testing this full message, I am not sure if there is a cleaner way than just using the snapshot test. Let me know if you'd much prefer instead to only test some part of the message and not use snapshots.